.github/maintainers_guide.md

```diff
@@ -71,25 +71,58 @@ If you make a mistake, don't fret. NPM allows you to unpublish a release within
+  * If you are releasing without learna, run `npm publish . --otp YOUR_OTP_CODE` in each of the package directories.
 
-3. Create GitHub Release(s) and add release notes.
+4. Create GitHub Release(s) and add release notes.
   * Release notes should mention contributors (@-mentions) and issues/PRs (#-mentions).
+  * Check the `This is a pre-release` checkbox.
+  * Example release: https://github.com/slackapi/node-slack-sdk/releases/tag/%40slack%2Fweb-api%405.11.0
 
-4. (Slack Internal) Communicate the release internally. Include a link to the GitHub Release(s).
+5. (Slack Internal) Communicate the release internally. Include a link to the GitHub Release(s).
 
-5. Announce on Bot Developer Hangout (`dev4slack.slack.com`) in **#slack-api**.
+6. Announce on Bot Developer Hangout (`dev4slack.slack.com`) in **#slack-api**.
 
-6. (Slack Internal) Tweet? Not necessary for patch updates, might be needed for minor updates, definitely needed for
+7. (Slack Internal) Tweet? Not necessary for patch updates, might be needed for minor updates, definitely needed for
    major updates. Include a link to the GitHub Release(s).
 
+### Releasing Beta Releases
+
+1. Make sure `main` has the latest changes from GitHub and rebase your branch from `main` by running `git rebase main` from your checked out branch. You can also `git merge main` if you are not comfortable with rebase.
+
+2. run the tests via `npm run test && npm run lint`
+
+3. update version in `package.json`. Use the existing version and add a hyphen to name the beta release (ex: `5.10.0-workflowStepsBeta.1`) (Note: the beta version must be in the format of `Major.Minor.Patch-BetaNamespace.BetaVersion`)
+
+4. Make a single commit for the version bump. The commit message should be similar to a [previous one](https://github.com/slackapi/node-slack-sdk/commit/1503609d79abf035e9e21bad7360e124e4211594).
+
+5. Git tag, push commit and tag up to origin
+  * Add a Git tag for every package you are releasing. example: `git tag @slack/web-api@5.10.0-workflowStepsBeta.1`
+  * Push all tags and commits up to the feature branch on `origin` via `git push origin feature-branch --tags`
+
+6. Publish to npm via `npm publish . --otp YOUR_OTP_CODE`
+  * We need to update the `dist-tags` on npm after a beta release. First update the `latest` `dist-tag` to the previous non beta release: `npm dist-tag add @slack/web-api@5.10.0 latest --otp YOUR_OTP_CODE`. Next, if you are adding/updating a feature/beta dist-tag, run `npm dist-tag add @slack/web-api@5.10.0-workflow-steps-beta.1 feat-workflow-steps --otp YOUR_OTP_CODE`
+
+7. Test by installing package from npm via `npm install @slack/web-api@feat-workflow-steps`
+
+8. Create GitHub Release(s) and add release notes.
+  * Release notes should mention contributors (@-mentions) and issues/PRs (#-mentions).
+  * Example release: https://github.com/slackapi/node-slack-sdk/releases/tag/%40slack%2Fweb-api%405.11.0
```

Comment:
Since this is for a beta release, should be mention ticking the pre-release checkbox?

Example:
* Check the `This is a pre-release` checkbox.